share/instruments/instruments.xml

```diff
@@ -13903,6 +13911,9 @@
                   <longName>Viola da gamba</longName>
                   <shortName>Vla. d. g.</shortName>
                   <description>Viola da gamba (tablature).</description>
+                  <StringData>
+                        <frets>13</frets>
+                  </StringData>
```

Comment:
This broke Bass Viol tablature. See https://musescore.org/en/node/376843 and #27380.
These 3 lines need to go, they do prevent the rest (and important part) of the string data to get 'inherited'